### PR TITLE
Reflect default experiment state (based on AMP_CONFIG) on experiments page

### LINKED ISSUE #7890: Reflect AMP_CONFIG experiment state on experiments page
The page on https://cdn.ampproject.org/experiments.html should reflect the AMP_CONFIG based experiment configuration

- Download https://cdn.ampproject.org/v0.js
- parse out AMP_CONFIG
- mark default-on experiments as `on by default` and disable the button

CC @dvoytenko, @rudygalfi

### PR DESCRIPTION
**Pull request by cramforce**
Fixes #7890

### CODE REVIEW

File: tools/experiments/experiments.js [hunk 1]
```diff
@@ -448,9 +450,41 @@ function showConfirmation_(message, callback) {
...
+    // Cache bust, so we immediately reflect AMP_CANARY cookie changes.
+    xhr.open('GET', '/v0.js?' + Math.random(), true);
+    xhr.send(null);
+  }).then(text => {
+    const match = text.match(/self\.AMP_CONFIG=([^;]+)/);
+    if (!match) {
+      throw new Error('Can\'t find AMP_CONFIG in: ' + text);
+    }
+    // Setting global var to make standard experiment code just work.
+    return self.AMP_CONFIG = JSON.parse(match[1]);
```
Comment (erwinmombay) [inline tools/experiments/experiments.js]: question, should we omit fields that we've forced to be ON?

Comment (cramforce) [inline tools/experiments/experiments.js]: I'm reconsidering. I thought we'd actually have the ability to turn them off, but that doesn't appear to be the case right now. So, adding `disabled` may make sense.

Comment (cramforce) [inline tools/experiments/experiments.js]: New look

screen shot 2017-03-23 at 10 52 42 am

Comment (erwinmombay) [inline tools/experiments/experiments.js]: awesome. thanks!

**Comment by erwinmombay** [comment]
@cramforce feel free to merge

**Pull request merged**